core/modules/widgets/widget.js

```diff
@@ -373,6 +373,21 @@ Widget.prototype.evaluateVariable = function(name,options) {
 	}
 };
 
+
+Widget.prototype.getSubstitutedText = function(text,options) {
```

Comment:
@Jermolene I will adapt once you have refactored the method for making a fake widget. This issue occurred to me while writing the code but I figured it made sense to just follow your lead on how you resolve the issue with the `makeFakeWidgetWithVariables()`, since I need to call that method here.